tests/testthat/test-duplicates.R

```diff
@@ -35,4 +39,10 @@ test_that("dataset of duplicate records can be accessed using `get_duplicates_da
   )
 
   expect_true(all(expected_ouput == get_duplicates_dataset()))
+
+
+  # This is where print.duplicates comes in play when run by the user
+  capture <- capture_output(get_duplicates_dataset(), print = TRUE)
+  expected_print_output <- "Duplicate records with respect to `USUBJID`."
+  expect_true(stringr::str_detect(capture, expected_print_output))
```

Comment:
Should we use the snapshot functionality mentioned by Daniel? Then we would check the complete output of the `print()` function.